include/sway/input/cursor.h

```diff
@@ -31,6 +31,13 @@ struct sway_cursor {
 	struct wl_listener button;
 	struct wl_listener axis;
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+=======
+>>>>>>> Register touch events in libtouch engine
+=======
+>>>>>>> b4eb3e10eeb8d6268490a6331571124027afd7ac
```

Comment:
doop

Reply:
Many doops throughout this commit

Reply:
Yeah, sorry about those. Something went terribly wrong here.